src/runtime_config/constitution.js

```diff
@@ -5,7 +5,101 @@ class Action {
   }
 }
 
+function toHexString(byteArray) {
+  console.log("To hex: " + byteArray.byteLength);
+  return Array.from(byteArray, function (byte) {
+    console.log("Byte: " + byte);
+    // return ("0" + (byte & 0xff).toString(16)).slice(-2);
```

Comment:
Needs fixing?

Reply:
No longer needed as we do the conversion from PEM to user ID in a wrapped C++ call.